docs/source/configuration/overview.mdx

```diff
@@ -139,7 +139,7 @@ If set, the Apollo Router runs in dev mode to help with local development.
 </td>
 <td>
 
-If set, the router watches for changes to its schema and configuration files and reloads them automatically without downtime.
+If set, the router watches for changes to its configuration file, and any supergraph file passed with `--supergraph` and reloads them automatically without downtime.  This only affects local files provided to the Router.  The supergraph and configuration provided from GraphOS via Launches (and delivered via Uplink) are _always_ loaded automatically, irregardless of this setting.
```

Comment:
```suggestion
If set, the router watches for changes to its configuration file and any supergraph file passed with `--supergraph` and reloads them automatically without downtime.  This setting only affects local files provided to the router.  The supergraph and configuration provided from GraphOS via Launches (and delivered via Uplink) are _always_ loaded automatically, regardless of this setting.
```

Reply:
I'll never stop making this mistake.  Thanks!